include/mrdox/Metadata/Symbols.hpp

```diff
@@ -25,7 +26,65 @@ namespace mrdox {
     USR. A USRs is a string that provide an
     unambiguous reference to a symbol.
 */
-using SymbolID = std::array<unsigned char, 20>;
+class SymbolID
+{
+    // std::uint8_t data_[20];
+    std::array<std::uint8_t, 20> data_;
```

Comment:
fuck, array again? why not `std::uint8_t data_[20];`?

Reply:
@vinniefalco `SHA1::hash` returns a `std::array<std::uint8_t, 20>`. Additionally, we extensively use `SymbolID` as a container (`operator[]`, `begin()`, `end()`, etc).

Reply:
Actually... I may be able to get rid of it... Let's see